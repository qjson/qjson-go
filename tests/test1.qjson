// test 1
a :Â b 

"c d": 
  `\n # multiline string
  the 
  brow
  fox
  ...
  `

e : [1,2,3]

f:{a:b,c:d}