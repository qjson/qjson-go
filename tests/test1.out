{"a":"b","c d":"the \nbrow\nfox\n...\n","e":[1,2,3],"f":{"a":"b","c":"d"}}
